jwst/ramp_fitting/ramp_fit.py

```diff
@@ -2946,13 +3166,15 @@ def calc_num_seg(gdq, n_int):
     gdq : float, 3D array
         cube of GROUPDQ array for a data
 
-    n_int : int
+    n_int : int (unused)
```

Comment:
If this is unused, is there a reason to keep it in the call signature?

Reply:
It's used in a test and I have not changed the test, yet.